lib/puppet/type/acl.rb

```diff
@@ -184,7 +195,7 @@ def insync?(is)
         s = p.tr '-', ''
         r << (s.sub!('r', '')?'r':'-')
         r << (s.sub!('w', '')?'w':'-')
-        r << (s.sub!('x', '')?'x':'-')
+        r << (s.sub!('x', '')?'x':'-')#'
```

Comment:
i.e. I should fix `ruby-mode` at some point, I guess, but then I'd have to learn ruby AND elisp. :D